frontend/demo/component/menubar/menu-bar-combo-buttons.ts

```diff
@@ -0,0 +1,45 @@
+import 'Frontend/demo/init'; // hidden-full-source-line
+import '@vaadin/flow-frontend/menubarConnector.js'; // hidden-full-source-line
+import '@vaadin/flow-frontend/contextMenuConnector.js'; // hidden-full-source-line
+
+import { html, LitElement, customElement, internalProperty } from 'lit-element';
+import '@vaadin/vaadin-menu-bar/vaadin-menu-bar';
+import '@vaadin/vaadin-icons/vaadin-icons';
+import { applyTheme } from 'Frontend/generated/theme';
+
+@customElement('menu-bar-basic')
+export class Example extends LitElement {
+  constructor() {
+    super();
+    // Apply custom theme (only supported if your app uses one)
+    applyTheme(this.shadowRoot);
+  }
+
+  // tag::snippet[]
+  @internalProperty()
+  private items = [
+    { text: 'Save' },
+    {
+      component: this.createItem(),
+      children: [{ text: 'Save as draft' }, { text: 'Save as copy' }, { text: 'Save and publish' }],
+    },
+  ];
+  // end::snippet[]
+
+  render() {
+    return html`
+      <!-- tag::snippethtml[] -->
+      <vaadin-menu-bar theme="primary" .items="${this.items}"></vaadin-menu-bar>
```

Comment:
```suggestion
      <vaadin-menu-bar theme="primary icon" .items="${this.items}"></vaadin-menu-bar>
```
Maybe adding an `icon` theme will make the buttons smaller and better looking.

Reply:
I was thinking about that, but that one is not a part of `menu-bar`, but getting it from `button` themes.
Not sure if we should add it here, but I guess it's not a problem 🤔

Reply:
I don’t see a problem. Menu Bar uses the button variants, "primary" is also coming from button I believe, not redefined in Menu Bar.